libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationViewModel.java

```diff
@@ -222,6 +224,15 @@ private void addNavigationListeners() {
     navigation.addFasterRouteListener(fasterRouteListener);
   }
 
+  private void addMilestones(NavigationViewOptions options) {
+    List<Milestone> milestones = options.milestones();
+    if (milestones != null && !milestones.isEmpty()) {
+      for (Milestone milestone : milestones) {
+        navigation.addMilestone(milestone);
```

Comment:
Have we ever considered adding an `addMilestones(List<Milestone>)` API to `MapboxNavigation`? That way we could use [`List#addAll`](https://docs.oracle.com/javase/7/docs/api/java/util/List.html#addAll(java.util.Collection)) for example and remove the _for each_ loop.

Reply:
@Guardiola31337 this is a great idea - will add, thanks!